### ISSUE TITLE
Conditional Display - Cloned Fields

### ISSUE THREAD

**Issue by nixnz** [rilwis/meta-box#637]
I recently purchased the Conditional Display add on however it is not working where a field has clone as being set to true. See my example below. I also raised this issue in  Wordpress Support but have not received a reply. Can you please provide a fix.

```php
array(
'name' => __( 'Extra Content', 'rwmb' ),
'id' => "{$prefix}extracontent",
'type' => 'select',

'options' => array(
''	=> 'None',
'siteadmin'	=> 'Site Admin',
'bookingscal'	=> 'iBex Booking Screen (calendar)',
'rooms'	=> 'iBex Room Details',
),
),

array(
'name' => __( 'Room ID', 'rwmb' ),
'id' => "{$prefix}rmID",
'type' => 'number',
'clone' => true,
'sort_clone' => true,
'hidden' => array('theme_meta_extracontent','!=','rooms'),
),
```

**Comment by rilwis** [comment]
Hi, this issue might be related to #636 (for cloned fields in group). We're checking and will add fix soon.

PS: We have support forum for customers where you can get support faster :)

**Comment by tanng** [comment]
Hi nixnz, 

Thanks for your reporting, we've released version 1.0.1 which fixed this bug. You can update and get problem solved now :)

Best regards

**Comment by nixnz** [comment]
Thanks for your very quick fix - it's working perfectly now :)
